crates/voicevox_core/src/internal.rs

```diff
@@ -134,14 +134,29 @@ impl Internal {
 
     //TODO:仮実装がlinterエラーにならないようにするための属性なのでこの関数を正式に実装する際にallow(unused_variables)を取り除くこと
     #[allow(unused_variables)]
-    pub fn voicevox_tts(
-        &self,
-        text: &CStr,
-        speaker_id: i64,
-        output_binary_size: *mut c_int,
-        output_wav: *const *mut u8,
-    ) -> Result<()> {
-        unimplemented!()
+    pub fn voicevox_tts(&mut self, text: &CStr, speaker_id: usize) -> Result<Vec<u8>> {
```

Comment:
```suggestion
    pub fn voicevox_tts(&mut self, text: &str, speaker_id: usize) -> Result<Vec<u8>> {
```

ここはCStrじゃなくても良さそうですね

Reply:
`&str` にしました！（ついでに `Internal::voicevox_load_openjtalk_dict` の引数の `&CStr` も `&str` に変えました）